mParticle-Example/MPKitExample.m

```diff
@@ -79,11 +98,26 @@ If your company SDK instance is available and is applicable (Please return nil i
     }
 }
 
+#pragma mark Identity
+- (MPKitExecStatus *)onIdentifyComplete:(FilteredMParticleUser *)user request:(FilteredMPIdentityApiRequest *)request {
+    [self updateUserWithMParticleUser:user];
+    return [self execStatus:MPKitReturnCodeSuccess];
+}
+
+- (MPKitExecStatus *)onLoginComplete:(FilteredMParticleUser *)user request:(FilteredMPIdentityApiRequest *)request {
+    [self updateUserWithMParticleUser:user];
+    return [self execStatus:MPKitReturnCodeSuccess];
+}
+
+- (MPKitExecStatus *)onModifyComplete:(FilteredMParticleUser *)user request:(FilteredMPIdentityApiRequest *)request {
+    [self updateUserWithMParticleUser:user];
+    return [self execStatus:MPKitReturnCodeSuccess];
+}
 
 #pragma mark Application
 /*
```

Comment:
I don't think we have any use here for any application lifecycle methods, this can all be removed

Reply:
👍